GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -66,6 +66,7 @@
 import org.waterforpeople.mapping.app.gwt.client.surveyinstance.SurveyInstanceDto;
 import org.waterforpeople.mapping.app.web.dto.SurveyRestRequest;
 import org.waterforpeople.mapping.dataexport.service.BulkDataServiceClient;
+import org.waterforpeople.mapping.serialization.response.MediaResponse;
```

Comment:
Is there any reason for which it wouldn't be added to the jar?
What's wrong with having this dependency?

I may be missing something of this build process...